content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,296 @@
+ 400         | `includeAssociatedData`, `_outputFormat`, `_since`, and `_type` contain invalid or unsupported values | error    | invalid
+ 400         | The resources provided in `_typeFilter` do not match the resources supplied in `_type`                | error    | invalid
+ 400         | `_typeFilter` contains multiple queries for the same resource                                         | error    | invalid
+ 400         | Unsupported parameters are passed to the export request                                               | error    | invalid
+ 422         | `_typeFilter` is provided with an invalid or unsupported value                                        | error    | invalid
+ 429         | There is already an export job in-progress for the client and tenant combination                      | error    | throttled
+
+## Status Request
+
+Checks the status of the export job.
+Use the URL returned in the `Content-Location` header from the kick off request to retrieve the status of a bulk data export:
+
+    GET  <URL from Content-Location header>
+
+_Notes_
+
+* Clients should follow an Exponential Backoff approach when polling for status as outlined in the [bulk data export specification.](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#bulk-data-status-request)
+
+### Authorization Types
+
+<%= authorization_types(provider: false, patient: false, system: true) %>
+
+_Notes_
+
+* Scopes are not checked in this request. The authorization token must be valid, and the client in the token must match the client that requested the export.
+
+### Headers
+
+<%= headers head: {'Accept': 'application/fhir+json', Authorization: '&lt;OAuth2 Bearer Token>'} %>
+
+### Example
+
+#### In-Progress Response
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+Response
+
+    Status: 202 Accepted
+    X-Progress: in-progress
+    Retry-After: 600
+
+#### Complete Response
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+Response
+
+    Status: 200 OK
+    Expires: Mon, 22 Jul 2025 23:59:59 GMT
+    Content-Type: application/json
+
+<%= json(:R4_BULK_EXPORT_COMPLETE) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## File Retrieval Request
+
+Once an export is complete, the status request will return file URLs which may be used to download the exported resources.
+
+    GET <File URL from status response>
+
+_Implementation Notes_
+
+* The file retrieval request returns a '307 Temporary Redirect' with a 'Location' header, the Location header must be followed in order to download the file
+* An exported ndjson file contains only a single resource type and there can be multiple files for the same resource type. There is no limit number of resources per file.
```

Comment:
we talked about it at some point, we might want to also add the expected max size for a file

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/pull/899/commits/4f1c2bf9f4ce582e4a76799ebb75b5c0ad282abe